src/components/AvatarGroup/counter.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import StyledCounter from './styled/counter';
+import abbreviateNumber from './helpers/abbreviateNumber';
+
+export default function Counter(props) {
+    const { avatars, size } = props;
+    const total = avatars.length;
+
+    return (
+        <StyledCounter size={size}>
+            <span>{abbreviateNumber(total)}</span>
```

Comment:
Use a more semantic tag, if we are going to show a text, use an `h1` instead `span`